src/Laravel/DbExtractor.php

```diff
@@ -19,6 +19,9 @@ class DbExtractor extends AbstractExtractor
     /** @var string */
     protected $updateColumn = 'updated_at';
 
+    public $statistics = [
```

Comment:
public because we do not inherit this class in helios, but just use it, and need to have a way to write data into statistics array. I can create method "add statistic info" for each place where we have $statistics defined to make it look nicer. - let me know if i should.